Makefile

```diff
@@ -116,4 +73,91 @@ docker_build: _base_docker_cntr
 		alpine -R $(shell whoami | id -u): /copy
 	@echo "Built binaries to $(OUT_D)"
 	@echo "Created a local Docker container. To use, run: docker run --rm -it doctl_local"
-# end docker targets
+
+.PHONY: test
+test:
+	@echo "==> running tests"
+	@echo ""
+	go test ./commands/... ./do/... ./pkg/... .
+
+.PHONY: shellcheck
+shellcheck:
+	@echo "==> analyzing shell scripts"
+	@echo ""
+	@scripts/shell_check.sh
+
+.PHONY: mocks
+mocks:
+	@echo "==> updating mocks"
+	@echo ""
+	@scripts/regenmocks.sh
+
+.PHONY: vendor
+vendor:
+	@echo "==> vendoring dependencies"
+	@echo ""
+	go mod vendor
+	go mod tidy
+
+.PHONY: clean
+clean:
+	@echo "==> removing build / release artifacts"
+	@echo ""
+	@rm -rf builds
+
+.PHONY: _install_github_release_notes
+_install_github_release_notes:
+	@GO111MODULE=off go get -u github.com/buchanae/github-release-notes
+
+.PHONY: _changelog
+_changelog: _install_github_release_notes
+	@scripts/changelog.sh
+
+.PHONY: changelog
+changelog: _install_github_release_notes
+	@echo "==> generating changelog"
+	@echo ""
+	@changes=$(shell scripts/changelog.sh) && cat $$changes && rm -f $$changes
+
+# ORIGIN ?= origin
+ORIGIN ?= hilary
+
+BUMP ?= patch
+
+.PHONY: version
+version:
+	@echo "==> determining doctl version"
+	@echo ""
+	@ORIGIN=$$ORIGIN scripts/version.sh
```

Comment:
This syntax didn't work for me during my local testing, I had to use,

```Makefile
        @ORIGIN=${ORIGIN}
```

And then it all worked great.